components/category_menu_items/category_menu_items.tsx

```diff
@@ -83,11 +83,18 @@ const CategoryMenuItems = (props: Props): JSX.Element | null => {
     }
 
     const categoryMenuItems = filteredCategories.map((category: ChannelCategory) => {
+        let text = category.display_name;
+        if (category.type === CategoryTypes.FAVORITES) {
+            text = intl.formatMessage({id: 'sidebar.types.favorites', defaultMessage: 'Favorites'}).toLowerCase();
```

Comment:
nit : may be we can add a small comment here saying since css applies `capitalize` just to be more clear to others on why the lowercase was used